scss/_variables.scss

```
@@ -1091,7 +1091,9 @@ $input-line-height-lg:                  $h5-line-height !default; // Boosted mod
 
 $input-transition:                      border-color $transition-duration $transition-timing, $transition-focus !default;
 
-$form-color-width:                      3rem !default;
+$form-color-width:                      2.5rem !default; // Boosted mod
```

```suggestion
$form-color-width:                      2.5rem !default; // Boosted mod: instead of `3rem`
```

Experimenting with the latest commits from Bootstrap this way of indicating whether a value has been changed from Bootstrap or is specific to Boosted.